packages/ember-glimmer/lib/syntax/render.js

```diff
@@ -186,8 +187,9 @@ class NonSingletonRenderManager extends AbstractRenderManager {
   create(environment, definition, args, dynamicScope) {
     let { name, env } = definition;
     let modelRef = args.positional.at(0);
+    let controllerFactory = env.owner[FACTORY_FOR](`controller:${name}`);
```

Comment:
Doesn't using factoryFor here mean the double extended class is never used for `render`, regardless of the feature flag setting?